back/engines/commercial/analytics/lib/tasks/analytics_tasks.rake

```diff
@@ -51,6 +51,29 @@ namespace :analytics do
       end
     end
   end
+
+  desc 'Populate locale dimensions from tenant config'
```

Comment:
I'm a bit uncomfortable with the idea of populating the dimensions using the rake task while the list of locales is likely to change.

Reply:
It possibly might be better running as a job, but I think we should leave it running with migrations for now as then at least we know the dimensions will get updated every time we release. Can we put that as a task for the future?